Modules/Components/itkImageRegistrationMethodv4/include/selxItkImageRegistrationMethodv4Component.hxx

```diff
@@ -241,7 +241,7 @@ ItkImageRegistrationMethodv4Component< Dimensionality, TPixel, InternalComputati
     movingIntensityMaximumCalculator->ComputeMaximum();
 
     MovingInvertIntensityImageFilterPointer movingIntensityInverter = MovingInvertIntensityImageFilterType::New();
-    movingIntensityInverter->SetInput(this->m_FixedImage);
+    movingIntensityInverter->SetInput(movingImage);
```

Comment:
I feel a bit confused about the subject: "BUG: Use function local image variable, not member variable". Is the bug about using member variables, or is it about using the moving versus using the fixed image?

Reply:
Both :)